app/views/forms/_add_element.slim

```diff
@@ -1,6 +1,6 @@
 = render partial: 'shared/error_messages', locals: {target: form_element }
 
-= form_for [form, form_element], remote: true, html: {class: 'form-element'}  do |f|
+= form_for [form, form_element], remote: true, html: {class: 'form-element', id: 'new_collection_element'}  do |f|
```

Comment:
ah interesting. In general I agree. I did that because it previously had an auto-generated id `new_form_element` courtesy of the form builder, and I just changed `form` to `collection`. There's a case for consistency with that I think, but also for consistency with the rest of our classes. I don't really care - should I change it?